### ISSUE TITLE
Konami Penthouse - Network error

### ISSUE THREAD

**Issue by TheOnlyWiseJEDI** [DestinationHome/Destination-Home-Online#61]
**Describe the bug**
received network error M-10 [D3] on first attempt to visit, second attempt was successful

**To Reproduce**
Attempt to load

# Client Information:
**Region:** SCEA
**Country:** 
**Platform:** RPCS3
**Firmware Type:** OFW

**Comment by NagatoDEV** [comment]
Is this a constant bug you experience when you go to Konami Penthouse?

**Comment by TheOnlyWiseJEDI** [comment]
I was using RPCS3 when I got that (network error M-10 [D3]) and it happened
a time or two throughout the day however by the end of the day I was
getting that on every attempt to visit any location I had not visited yet.
No matter what space I tried to visit I got the error over and over. Once
the error appeared and I got bumped out of home it reloaded and I was able
to visit the space in question with no trouble. Restart of RPCS3 and
deleting home cache had no effect.

Ron (Jedi)
